kivy/core/video/video_ffpyplayer.py

```diff
@@ -115,6 +115,11 @@ def __init__(self, **kwargs):
 
         super(VideoFFPy, self).__init__(**kwargs)
 
+    @property
+    def _is_stream(self):
+        # XXX: tested with RTSP streams only right now
+        return self.filename.startswith('rtsp://')
```

Comment:
_is_stream is only used when building ff_opts, to prevent starting player paused and can probably be removed as soon as the 'eof' receiving issue is solved

Reply:
Commented the purpose of _is_stream